main.cpp

```diff
@@ -90,8 +90,88 @@ good to go!
 
  Wait for my code review.
  */
-
 #include <iostream>
+
+struct FloatType
+{
+    float add( float lhs, float rhs )
+    {
+        return lhs + rhs;
+    }
+    float subtract( float lhs, float rhs )
+    {
+        return lhs - rhs;
+    }
+    float multiply( float lhs, float rhs )
+    {
+        return lhs * rhs;
+    }
+    float divide(float lhs, float rhs)
+    {
+        if( rhs > 0 );
+        else if( rhs < 0 );
+        else
+        {
+            std::cout << "\nwarning, floating point division by zero returns 'inf' !\n";
+        }
```

Comment:
`if( rhs == 0.f )` 
- [ ] revise

Reply:
I thought I got an error about using == with floats last time... glad to be corrected